.ado/xcconfig/publish_overrides.xcconfig

```diff
@@ -1,10 +1,10 @@
 // For publish builds, only provide line tables for symbolizing crashes
-CLANG_DEBUG_INFORMATION_LEVEL=line-tables-only
+CLANG_DEBUG_INFORMATION_LEVEL[config=Release]=line-tables-only
```

Comment:
Why only release?

Reply:
Because we want the default level of debug information for the Debug configuration.  For debugging.

Reply:
but that will make our nugget size for debug build much bigger which the size exception approvers won't like. can we have a different script or a setting that it is only used for local nugget package build but leave the CI alone?

Reply:
Our NuGet is ship only.  This doesn't affect the package size.